grr/server/grr_response_server/gui/ui/lib/models/model_test_util.ts

```diff
@@ -96,3 +96,23 @@ export function newClientApproval(args: Partial<ClientApproval> = {}):
     ...args,
   };
 }
+
+/**
+ * A helper function to construct a FlowResultSet object.
+ * Feel free to extend it with more default arguments.
```

Comment:
"Feel free to extend it with more default arguments." is a comment about an invariant that might not age well. I suggest removing it.